sw/device/silicon_creator/lib/manifest.h

```diff
@@ -18,6 +18,59 @@
 extern "C" {
 #endif  // __cplusplus
 
+/**
+ * Usage constraints.
+ *
+ * This struct is used to constrain a boot stage image to a set of devices based
+ * on their device IDs, creator and/or owner manufacturing states, and life
+ * cycle states. Bits of `selector_bits` determine which fields (or individual
+ * words of a field as in the case of `device_id`) must be read from the
+ * hardware during verification. Unselected fields must be set to
+ * `MANIFEST_USAGE_CONSTRAINT_UNSELECTED_WORD_VAL` to be able to generate a
+ * consistent value during verification.
+ */
+typedef struct manifest_usage_constraints {
+  /**
+   * Usage constraint selector bits.
+   */
+  uint32_t selector_bits;
+  /**
+   * Device identifier value which is compared against the `DEVICE_ID` value
+   * stored in the `HW_CFG` partition in OTP.
+   *
+   * Mapped to bits 0-8 of `selector_bits`.
```

Comment:
I think we'd normally write this as bits 0-7.

Reply:
Yes, I would too :) This is a typo, fixed locally but apparently I forgot to push the change.